roles/keystone/tasks/federation.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: install mod_auth_openidc deps
+  apt: name={{ item }} state=present
+  with_items:
+    - libjansson4
+    - libhiredis0.10
+    - libcurl3
+
+- name: download mod_auth_openidc
+  get_url: url={{ keystone.federation.sp.oidc.download.url }}
+           dest=/tmp/libapache2-mod-auth-openidc_{{ keystone.federation.sp.oidc.version }}-1_amd64.deb
+
+- name: install mod_auth_openidc
+  apt: deb=/tmp/libapache2-mod-auth-openidc_{{ keystone.federation.sp.oidc.version }}-1_amd64.deb
```

Comment:
Why go to the length of defining an oidc version variable, only to have it incomplete? If the version changes or the package build changes we'll have to edit things in two places. It seems the variable is only used in these lines here, so I'm not sure what value the variable is bringing.